sssom/validators.py

```diff
@@ -83,7 +83,7 @@ def check_all_prefixes_in_curie_map(msdf: MappingSetDataFrame) -> None:
 
     missing_prefixes = []
     for pref in prefixes:
-        if pref not in list(msdf.prefix_map.keys()):
```

Comment:
Out of curiosity (not a blocker for merging) but shouldn't we prefent an empty string being added to the `prefixes` variable in the first place?

Reply:
You're spot on! Was missing an `if` in `util.py`. [Added it now](https://github.com/mapping-commons/sssom-py/pull/308/commits/6fd501fb8b5be197d27a9c467400a8275364bcf0)!